tools/build/build.sh

```diff
@@ -28,7 +28,7 @@ mkdir -p "$ARTIFACTS_OUT"
 
 npm install --quiet
 
-for version in 10.0.0 12.0.0 14.0.0 15.0.0 16.0.0
+for version in 12.0.0 14.0.0 16.0.0
```

Comment:
Is there an issue for node-pre-gyp to support Node 18 already filed?

Reply:
No. Actually, let me file those issues.

Reply:
Thanks!

Reply:
Looks like node 17 is already supported, but our package-lock.json is fixed on an older version of node-pre-gyp. I'll update our package.json file, so 17 can be included.

The test would then need to be version 17.0.1, because 17.0.0 had [issues](https://github.com/JoshuaWise/better-sqlite3/issues/711).

For node 18, I filed an [issue](https://github.com/mapbox/node-pre-gyp/issues/647).